lib/appsignal/transaction.ex

```diff
@@ -63,7 +63,7 @@ defmodule Appsignal.Transaction do
   `Appsignal.TransactionRegistry`.
 
   """
-  @spec start(String.t, String.t) :: Transaction.t
+  @spec start(String.t, atom) :: Transaction.t
```

Comment:
would we maybe want to allow a Sting type here, and use remove the `Atom.to_string(namespace)` on the line below? Or maybe always call `to_string(namespace)` on it and allow both Atoms and Strings?

Not sure if changing the type would break things for people who now pass in Strings, or would that not work at all?

Reply:
The type spec makes our choice to use an atom explicit, and will allow us to check to make sure the library is calling its own functions properly. The type specs don't have an effect on the code itself. 

We've always expected an atom here (passing a string to `Atom.to_string/1` will crash), the type spec was just incorrect, causing a lot of Dialyzer warnings.